go-controller/pkg/types/const.go

```diff
@@ -36,6 +36,19 @@ const (
 	DirectionToLPort   = "to-lport"
 	DirectionFromLPort = "from-lport"
 
+	// ACL Priorities
+
+	// Default routed multicast allow acl rule priority
+	DefaultRoutedMcastAllowPriority = "1013"
+	// Default multicast allow acl rule priority
+	DefaultMcastAllowPriority = "1012"
+	// Default multicast deny acl rule priority
+	DefaultMcastDenyPriority = "1011"
+	// Default allow acl rule priority
+	DefaultAllowPriority = "1001"
+	// Default deny acl rule priority
+	DefaultDenyPriority = "1000"
```

Comment:
Consider better naming; none of these are really "Default"s since they can't be overridden. Also, the Allow/Deny priorities are specifically for Allow/Deny rules for NetworkPolicy, not all Allows/Denies.

Reply:
I only moved these to another file, and though I agree that they could use a better naming convention I would like to avoid doing it in this PR as to not delay it further.